src/hub.js

```diff
@@ -334,6 +341,7 @@ document.addEventListener("DOMContentLoaded", async () => {
   }
 
   const scene = document.querySelector("a-scene");
+  scene.object3D.autoUpdate = false;
```

Comment:
^^ note other major change. this results in killing the world matrix update loop in three.js, which allows us to run it ourselves in the aframe system.

(incidentally, I'm not sure if this is totally necessary now, but I like having this loop under our control atm.)